src/main/java/edu/hillel/homework/lesson21/dao/HomeworkDao.java

```diff
@@ -0,0 +1,90 @@
+package edu.hillel.homework.lesson21.dao;
+
+import edu.hillel.homework.lesson21.dto.Homework;
+import org.jetbrains.annotations.NotNull;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomeworkDao {
+
+    public void addHomework(@NotNull Homework homework) {
+        try (Connection connection = DataBaseConnection.getConnection();
```

Comment:
Please, send connection via constructor, the HomeworkDao shouldn't know about DataBaseConnection, it should be incapsulated

Reply:
Well done.